src/model/APNSPriority.php

```diff
@@ -4,4 +4,13 @@
 class APNSPriority {
 	public const IMMEDIATE = 10;
 	public const THROTTLED = 5;
+
+	public static function isValid( int $key ): bool {
+		return in_array(
```

Comment:
I'm surprised by this function using camel case, but `in_array` using snake case. 

Is this a convention like standard lib / free functions use snake case, the others use camel case?

Reply:
Ooof, yeah, welcome to PHP – the standard library is _very_ unfortunately named. 

Typical convention for classes is camelCase though – and the linter seems to like it!

Reply:
🐫 👍